src/platyPS/platyPS.psm1

```diff
@@ -641,6 +649,10 @@ function New-ExternalHelp
                 $value = $r.AboutMarkDownToString($model)
 
                 $outPath = Join-Path $OutputPath ([io.path]::GetFileNameWithoutExtension($About.FullName) + ".txt")
+                if((Split-Path -Leaf $outPath).ToUpper().Substring(0,6) -ne "ABOUT_")
```

Comment:
It's better to replace it by `(Split-Path -Leaf $outPath).StartsWith('about_', $true, $null)`